### ISSUE TITLE
Pretrained Weights and Dataset Splits

### ISSUE THREAD

**Issue by YAD07** [SonyResearch/RAW-Diffusion#6]
Dear Author，

I hope this message finds you well. I’m reaching out because I’ve been working on reproducing the results of your paper using the code and dataset provided in your GitHub repository.
1.​​Pretrained Model Weights​​
Would it be possible for you to share the pretrained model weights used in your experiments? This would help ensure that my implementation aligns with your setup.
​​2.Dataset Splits​​
I noticed that the dataset splits (train/test) are generated using train_test_split with a fixed random seed (seed=2817). However, my reproduced results are significantly higher than those reported in your paper. Could you clarify.Whether the exact split indices (or a list of filenames for train/test) are available?

**Comment by shayloshaylolo-creator** [comment]
Сделай эту девушку лежащей на траве.Из груди торчит кровавый нож,а по всюду кровь.Трава темного зелёного цвета,вся в крови.Вид должен быть сверху. По бокам стоят люди,которые в панике и кто то уже плачет. (Но их не особо видно)

<img width="1024" height="1024" alt="Image" src="https://github.com/user-attachments/assets/6c026263-96b8-421e-8e5b-da529bff6554" />
